integration_tests/test_endpoints.py

```diff
@@ -237,7 +236,9 @@ def test_sync_streaming_model_endpoint(capsys):
             delete_model_endpoint(create_endpoint_request["name"], user)
 
 
-@pytest.mark.skipif(CIRCLECI, reason="skip on circleci since need to figure out s3 access")
+@pytest.mark.skipif(
```

Comment:
@saiatmakuri FYI, i don't think this sounds as an integration test. can consider checking # tokens from responses?

Reply:
yeah can skip for now. essentially want to check if every tokenizer can be made before each deploy

Reply:
it currently fails for me since initdb is not in bash container. can be fixed by adding postgres there